docs/track.rst

```diff
@@ -225,7 +225,10 @@ The ``corpora`` section contains all document corpora that are used by this trac
 
 Each entry in the ``documents`` list consists of the following properties:
 
-* ``base-url`` (optional): A http(s) or S3 URL that points to the root path where Rally can obtain the corresponding source file. Rally can also download data from private S3 buckets if access is properly `configured <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration>`_.
+* ``base-url`` (optional): A http(s), S3 or GS URL that points to the root path where Rally can obtain the corresponding source file. Rally can also download data from private S3 or GS buckets if access is properly configured:
```

Comment:
typo: `GS` -> `GCS`

Reply:
The scheme is gs not gcs (i.e. `gs://bucket`). Also the service itself is called Google Storage (https://cloud.google.com/storage).

Reply:
While Amazon calls the product "S3", the official name of the equivalent Google service seems to be "Cloud Storage" according to the docs you point to. Should we then refer to it by that name instead of a different acronym? Or is "GS" the official acronym (I did not find it in Google's documentation)?

Reply:
Standardizing on `Google Storage` and fixed in 27419b61228b130511f2ced2e08f6bc32ac37a42